pkgs/development/python-modules/llfuse/default.nix

```diff
@@ -26,21 +26,19 @@ buildPythonPackage rec {
   };
 
   nativeBuildInputs = [ cython pkg-config ];
-
   buildInputs = [ fuse ];
-
   propagatedBuildInputs = [ contextlib2 ];
+  checkInputs = [ pytestCheckHook which ];
```

Comment:
Don't move around random stuff.

Reply:
This was the precise location of the merge conflict. I should've checked the resulting diff more closely.

Reply:
Fixed.